zsh/configs/post/path.zsh

```diff
@@ -1,8 +1,10 @@
 # ensure dotfiles bin directory is loaded first
 PATH="$HOME/.bin:/usr/local/sbin:$PATH"
 
-# load rbenv if available
-if command -v rbenv >/dev/null; then
+# load ASDF, falling back to rbenv if not available
+if [ -d "$HOME/.asdf" ]; then
+  . $HOME/.asdf/asdf.sh
```

Comment:
Completions and zsh initialization order are the perfect storm of my dotfile nightmares. I think we can move forward without it and tackle later if someone is sufficiently motivated.